? admin_role_drush_mm_integration.module
? admin_role_drush_mm_integration.patch
Index: adminrole.module
+===================================================================
RCS file: /cvs/drupal-contrib/contributions/modules/adminrole/adminrole.module,v
retrieving revision 1.2.2.8.2.6
diff -u -p -r1.2.2.8.2.6 adminrole.module
--- adminrole.module   7 Jan 2009 21:12:03 -0000       1.2.2.8.2.6
+++ adminrole.module   21 Feb 2009 23:49:19 -0000
@@ -42,6 +42,13 @@ function adminrole_menu() {
   return $items;
}
 
+/**
+ * Implementation of hook_drush_mm_enable().
+ */
+function adminrole_drush_mm_enable($module) {
+  adminrole_update_perms();
+}
+
 function adminrole_update_perms() {
   if ($admin_role = variable_get('adminrole_adminrole', 0)) {
     if ($admin_role == 0) {
